pkg/cloud/services/ec2/routetables.go

```diff
@@ -317,6 +317,8 @@ func (s *Service) getRouteTableTagParams(id string, public bool) infrav1.BuildPa
 	} else {
 		name.WriteString("private")
 	}
+	name.WriteString("-")
+	name.WriteString(zone)
```

Comment:
Do we need to worry about tag maximum length?

Reply:
The maximum length for the value is 256 Unicode characters (https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html).

I could add a check and fail if it exceeds this length?

Reply:
@vincepri - i thought we could perhaps create a new issue for checking length of labels as it applies to quite a few places....not just the routetable and subnets that i have added to. It could be a good first issue for someone?

Reply:
Yeah, let's create an issue for this.

Reply:
Created #1746